front/src/Phaser/Map/ITiledMap.ts

```diff
@@ -78,6 +78,8 @@ export interface ITiledMapTileLayer {
     width: number;
     x: number;
     y: number;
+    parallaxx: number;
+    parallaxy: number;
```

Comment:
You should probably make those optional. It is making the tests fail.

Reply:
Ah, good catch! Sure